src/plugin.json

```diff
@@ -48,18 +55,12 @@
       "type": "dashboard",
       "name": "Lots of Stats",
       "path": "dashboards/stats.json"
-    },
-    {
-      "type": "page",
-      "name": "Angular Page",
-      "component": "AngularExamplePageCtrl",
-      "role": "Viewer",
-      "addToNav": true
     }
   ],
 
   "dependencies": {
-    "grafanaVersion": "6.6.x",
+    "grafanaDependency": ">=7.0.0",
+    "grafanaVersion": "7.0.0",
```

Comment:
Just out of curiosity - why is the version specified in two ways?

Reply:
The short answer is so that it satisfies the [plugin schema](https://github.com/grafana/grafana/blob/master/docs/sources/developers/plugins/plugin.schema.json).

While `grafanaDependency` needs to be a valid semver format, `grafanaVersion` isn't really enforced by anything other than the schema. 

I'd make `"grafanaVersion": ">=7.0.0",`, but that would require a schema update.

Reply:
Hm, am I missing something? In the schema, it only says `"required": ["grafanaDependency"],`, so that shouldn't it be sufficient?
(Also seems like the reasonable approach to me => semver with the new property and the old property is deprecated and not required any more.)

Reply:
Theoretically, yes. But Grafana.com hasn't yet updated to show `grafanaDependency` for plugin pages, so it's not _really_ deprecated yet.

Reply:
Ahhh, gotcha.